src/_pytest/mark/structures.py

```diff
@@ -298,7 +299,7 @@ def test_function():
     on the ``test_function`` object. """
 
     _config = None
-    _markers = set()
+    _markers: Set[str] = set()
```

Comment:
this is python3.6+ syntax, we can't use this yet (still support python 3.5) -- you'll need type comments here

Reply:
Fixed, added `python_version = 3.5` which catches these things.

Reply:
assumedly our testsuite should catch that as well so you shouldn't need that setting :thinking: